### PR TITLE
Added consistent spacing in documentation comments. [#159798567]

### PR DESCRIPTION
**Pull request by RandallValentin**

### CODE REVIEW

File: src/ServiceHub.User.Context/Models/Name.cs [hunk 1]
```diff
@@ -12,12 +12,12 @@ namespace ServiceHub.User.Context.Models
     /// </remarks>
     public class Name
     {
-        /// <value>The unique Id for the user's Name Object</value>
+        /// <value>The unique Id for the user's Name Object.</value>
         [BsonId]
         public Guid NameId { get; set; }
-        /// <value>The user's first name(s).</value>
+        /// <value>The user's first name(s). </value>
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Models/Name.cs]: Remove space

File: src/ServiceHub.User.Context/Models/Name.cs [hunk 2]
```diff
@@ -12,12 +12,12 @@ namespace ServiceHub.User.Context.Models
...
         public string First { get; set; }
-        /// <value>The user's middle name(s).</value>
+        /// <value>The user's middle name(s). </value>
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Models/Name.cs]: Remove the space

File: src/ServiceHub.User.Context/Utilities/UserModelMapper.cs [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
-        /// Convert a Context-User model to a Library-User model 
+        /// Convert a Context-User model to a Library-User model. 
         /// </summary>
-        /// <remarks>Use this mapper to convert an object retrieved from the 
-        /// Context data source into a Library model to return to the client</remarks>
-        /// <param name="contextUser">The user model to convert</param>
-        /// <returns>A Library User model, or null if the input Context User was null</returns>
+        /// <remarks>
+        /// Use this mapper to convert an object retrieved from the 
+        /// Context data source into a Library model to return to the client.
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Utilities/UserModelMapper.cs]: Change Context to full project name for clarity

File: src/ServiceHub.User.Context/Utilities/UserModelMapper.cs [hunk 4]
```diff
@@ -55,14 +59,15 @@ public static Library.Models.User ContextToLibrary(Context.Models.User contextUs
         }
 
         /// <summary>
-        /// Convert the Library-User model to a Context-User model
+        /// Convert the Library-User model to a Context-User model.
         /// </summary>
         /// <remarks>
-        /// Convert models with this method before trying to use them to
-        /// interact with the Context data source.
+        /// Convert models with this method before trying to use them to interact with the Context data source.
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Utilities/UserModelMapper.cs]: Change Context to full project name for clarity

File: src/ServiceHub.User.Context/Utilities/UserModelMapper.cs [hunk 5]
```diff
@@ -55,14 +59,15 @@ public static Library.Models.User ContextToLibrary(Context.Models.User contextUs
...
         /// </remarks>
-        /// <param name="libraryUser">The library user to convert to a context user</param>
-        /// <returns>A Context User if the Library User was valid, null otherwise</returns>
+        /// <param name="libraryUser">The library user to convert to a context user.</param>
+        /// <returns>
+        /// A Context User if the Library User was valid, null otherwise.
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Utilities/UserModelMapper.cs]: Change library to full project name for clarity

File: src/ServiceHub.User.Context/Utilities/UserModelMapper.cs [hunk 6]
```diff
@@ -106,11 +111,13 @@ public static Context.Models.User LibraryToContext(Library.Models.User libraryUs
 
 
         /// <summary>
-        /// Validates and maps a list of Context Users to Library Users
+        /// Validates and maps a list of Context Users to Library Users.
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Utilities/UserModelMapper.cs]: see previous comment

File: src/ServiceHub.User.Context/Utilities/UserModelMapper.cs [hunk 7]
```diff
@@ -106,11 +111,13 @@ public static Context.Models.User LibraryToContext(Library.Models.User libraryUs
...
         /// </summary>
-        /// <param name="contextUsers">The users to validate and map</param>
-        /// <returns>A list of Library Users if all Context users were valid, and null
-        /// if one or more users were invalid.</returns>
+        /// <param name="contextUsers">The users to validate and map.</param>
+        /// <returns>
+        /// A list of Library Users if all Context users were valid,
```
Comment (ajzoell) [inline src/ServiceHub.User.Context/Utilities/UserModelMapper.cs]: See previous comments

File: src/ServiceHub.User.Service/Controllers/UserController.cs [hunk 8]
```diff
@@ -53,7 +55,7 @@ public async Task<IActionResult> Get()
         /// Finds the user based on the id.
         /// </summary>
         /// <param name="id"></param>
-        /// <returns>the user with matching Id, or a 404 error</returns>
+        /// <returns>The user with matching Id, or a 404 error.</returns>
```
Comment (ajzoell) [inline src/ServiceHub.User.Service/Controllers/UserController.cs]: Id should be lowercase to match parameter name above

File: src/ServiceHub.User.Service/Controllers/UserController.cs [hunk 9]
```diff
@@ -258,11 +265,14 @@ public async Task<IActionResult> Put([FromBody]ServiceHub.User.Library.Models.Us
         /// <summary>
         /// Creates a new user.
         /// </summary>
-        /// <param name="user"> A User model to be provided from an external source
-        /// via JSON.  If the model is a valid model, it will be cast to a db-ready model
-        /// and stored in the database. </param>
-        /// <returns> If the user is accepted, it will return a 201, Created code.
-        /// Otherwise, it will return a 400, client-error code. </returns>
+        /// <param name="user">
+        /// A User model to be provided from an external source via JSON.
```
Comment (ajzoell) [inline src/ServiceHub.User.Service/Controllers/UserController.cs]: This should only tell me what the parameter is. Any description about what happens to the user should be in the summary, not the parameter line.

File: src/ServiceHub.User.Context/Models/Name.cs (hunk 1 above)
Comment (RandallValentin) [reply src/ServiceHub.User.Context/Models/Name.cs]: Done

File: src/ServiceHub.User.Context/Models/Name.cs (hunk 2 above)
Comment (RandallValentin) [reply src/ServiceHub.User.Context/Models/Name.cs]: Done

File: src/ServiceHub.User.Context/Utilities/UserModelMapper.cs (hunk 4 above)
Comment (RandallValentin) [reply src/ServiceHub.User.Context/Utilities/UserModelMapper.cs]: No context on this line. This line was left unchanged.

**Pull request merged**